Dockerfile

```
@@ -47,6 +48,9 @@ RUN apt-get -y --no-install-recommends install libboost-chrono1.83.0 liblua5.4-0
 COPY --from=chairemobilite/trrouting:latest /usr/local/bin/trRouting /usr/local/bin/
 COPY --from=greenscientist/osrm-backend:trixie /usr/local/bin/osrm-* /usr/local/bin/
 
+# For Rust binding
+RUN apt-get -y --no-install-recommends install cargo rustfmt rust-clippy
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)